Makefile

```diff
@@ -303,6 +304,10 @@ list-chartpress-images:
 	@rm chartpress.yaml
 	@mv chartpress.backup.yaml chartpress.yaml
 
+.PHONY: chartpress-cleanup
+chartpress-cleanup:
+	@chartpress --tag 0.1.0 --no-build
```

Comment:
question: We can just run the chartpress command to reset. There is a specific flag for that. So why not do that? I.e. you can run `chartpress --reset`.

Reply:
Done in f32f42bdafea58c794aa8eae23b758385cb58e1d.